modules/images/fetchpriority/load.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Module Name: Fetchpriority
+ * Description: Adds `fetchpriority=high` parameter to the non lazy loaded images.
+ * Experimental: Yes
+ *
+ * @since n.e.x.t
+ * @package performance-lab
+ */
+
+/**
+ * Filter image tags in content to add fetchpriority to the image tag if lazy loaded is not set.
+ *
+ * @since n.e.x.t
+ *
+ * @param string $filtered_image The filtered image.
+ * @param string $context        The context of the image.
+ *
+ * @return string image tag
+ */
+function fetchpriority_img_tag_add( $filtered_image, $context ) {
+
+	if ( 'the_content' !== $context ) {
+		return $filtered_image;
+	}
+
+	if ( strpos( $filtered_image, 'loading="lazy"' ) === false ) {
```

Comment:
We should make sure to have a space at the beginning of this string, to make sure it doesn't match something like `fooloading="lazy"`. Also, while that's unlikely to already be present, we must avoid adding the `fetchpriority` attribute if the image tag already has one on it, so let's include that here as well.
```suggestion
	if ( strpos( $filtered_image, ' loading="lazy"' ) === false && strpos( $filtered_image, ' fetchpriority=' ) === false ) {
```